content/en/blog/_posts/2023-01-17-confidential-kubernetes.md

```diff
@@ -0,0 +1,155 @@
+---
+layout: blog
+title: "Confidential Kubernetes: Use Confidential Virtual Machines and Enclaves to improve your cluster security."
+date: 2023-01-17
+slug: "confidential-kubernetes"
+---
+
+**Authors:** Fabian Kammel (Edgeless Systems), (more to add...)
```

Comment:
Noting that this is pending a fix

/retitle [WIP] Add blog post on Confidential Kubernetes

Reply:
I have added @mythi & @fitzthum as additional authors and removed the TODO.

Let me know (or propose a change) to add yourself as an author!